api/client/volume/client.go

```diff
@@ -435,3 +435,51 @@ func (v *volumeClient) Unquiesce(volumeID string) error {
 	}
 	return nil
 }
+
+func (v *volumeClient) CredsList() (map[string]interface{}, error) {
+	creds := make(map[string]interface{}, 0)
+	err := v.c.Get().Resource(credsPath + "/credslist").Do().Unmarshal(&creds)
```

Comment:
I'll skip these for the this review, because it looks like all APIs do this, but there should be a check on the returned HTTP status of the header. This is more of an overall question about all of OSD client calls.